Airport Ticket Booking System/BookingSystemUtils/InputValidation.cs

```diff
@@ -0,0 +1,93 @@
+﻿namespace Airport_Ticket_Booking_System;
+
+public class InputValidation : IInputValidation
+{
+    public ValidationErrorType IsValidFlightNumber(string flightNumber)
+    {
+        if (string.IsNullOrEmpty(flightNumber))
+        {
+            return ValidationErrorType.RequiredField;
+        }
+
+        if (flightNumber.Length < 7 || !char.IsLetter(flightNumber[0]) || !char.IsLetter(flightNumber[1]))
```

Comment:
you need to also check if `flightNumber` is long enough or `flightNumber[0]` and `flightNumber[1]` will throw an error.